projects/SEP_TEST/study_info/SEP_TEST_study_info_v1.json

```diff
@@ -0,0 +1,68 @@
+[
+    {
+        "field_name":"study_info_field_1",
+        "form_name":"sep_study_info",
+        "section_header":"",
+        "field_type":"info",
```

Comment:
Similarly, could this be set to `descriptive`?

Reply:
Yes could be, but unless there is html involved, we tend to use simple `info` type.